src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -77,8 +77,17 @@ public function render(PageBridge $content, $partial = false)
         $webspace = $this->webspaceManager->findWebspaceByKey($content->getWebspaceKey());
         $this->activeTheme->setName($webspace->getTheme()->getKey());
 
+        $query = [];
+        $request = [];
+        $cookies = [];
+        if ($currentRequest !== null) {
+            $query = $currentRequest->query->all();
```

Comment:
this is not possible for preview. therefor can do it only if there exists a current-request.